client/jetpack-cloud/sections/partner-portal/license-product-card/index.tsx

```diff
@@ -32,9 +33,10 @@ export default function LicenseProductCard( props: Props ) {
 		onSelectProduct,
 		suggestedProduct,
 		isMultiSelect,
+		showLicenseInfoModal,
 	} = props;
 	const productTitle = getProductTitle( product.name );
-	const [ showLightbox, setShowLightbox ] = useState( false );
+	const [ showLightbox, setShowLightbox ] = useState( showLicenseInfoModal );
```

Comment:
Are there reasons why we can't query the URL directly here to set the default state?

Reply:
Fixed here - https://github.com/Automattic/wp-calypso/pull/78850/commits/9a44723c4e9364c9294c3f63689b1e1d34bdb738